src/main/java/formflow/library/ScreenController.java

```diff
@@ -116,6 +116,11 @@ ModelAndView getScreen(
       return new ModelAndView("redirect:" + lockedSubmissionRedirectUrl);
     }
 
+    String redirectPath = getRedirectPath(flow, uuid, currentScreen, submission);
```

Comment:
I think this method name needs to be clearer -- it's not just getting a redirect path, it's checking if we have a condition on the page and run it if we do, returning the URL it thinks we should be on (running through all page level conditions in between to allow it to skip ahead to the next "runnable" page). 

What's a good name for that?   checkPageLevelCondition()... I don't have any good ideas that would reflect what it's doing and that it's returning a redirect Path (or not).   Is this one method doing too much?   Maybe you could do the check for the condition existing here and then call this getRedirectPath() since then it *will* be returning a path. 

Is this condition existing to aid in your development process, or will it run in live code as well?

Reply:
This is really great - thanks for creating this.

Reply:
Yeah - will come up with a better name. 

Yeah, it's a development process aid - we want to "disable" pages that have gone through acceptance, but maybe aren't staging/production ready.

Reply:
I like the re-org, I think it makes it a little clearer what's going on.